src/js/control-bar/audio-track-controls/audio-track-button.js

```diff
@@ -25,6 +25,8 @@ class AudioTrackButton extends TrackButton {
     options.tracks = player.audioTracks();
 
     super(player, options);
+
+    this.setIcon('vjs-icon-audio');
```

Comment:
If all the icons are prefixed with `vjs-icon-`, perhaps this should be automatically prepended, so we can simply call `this.setIcon('audio')`?